dvsb/metric/metric.py

```diff
@@ -11,7 +11,12 @@ def get_name(self) -> str:
         ...
 
     @abstractmethod
-    def compute(self, y_true: list[list[int]], scores: npt.NDArray[np.float64]) -> float:
+    def compute(
+        self,
+        y_true: list[list[int]],
+        scores: npt.NDArray[np.float64] | None = None,
```

Comment:
Using `|` for creating union is only supported Python >= 3.10 but I would like to support Python 3.9 as well.
Can you replace this with `Optional[npt.NDArray[np.float64]]`? (same goes for other parts)

Reply:
Oh, my bad, I'm too used to the >= 3.10 style of typing now! It's changed to `Optional[...]`.